### ISSUE TITLE
Windows Compile Issue?

### ISSUE THREAD

**Issue by jobenso** [clinton-hall/nzbToMedia#948]
The following was installed on windows 7 machine: pywin32-220.win32-py2.7, python-2.7 & pyinstaller-2.0

When I run this I get...

c:>C:\Python27\python.exe C:\pyinstaller-2.0\pyinstaller.py -F -w C:\nzbToMedia\nzbToMedia.py = single file, -w = suppresses console
Traceback (most recent call last):
  File "C:\pyinstaller-2.0\pyinstaller.py", line 91, in <module>
    main()
  File "C:\pyinstaller-2.0\pyinstaller.py", line 84, in main
    spec_file = run_makespec(opts, args)
  File "C:\pyinstaller-2.0\pyinstaller.py", line 44, in run_makespec
    spec_file = PyInstaller.makespec.main(args, **opts.**dict**)
  File "C:\pyinstaller-2.0\PyInstaller\makespec.py", line 302, in main
    specfile = open(specfnm, 'w')
IOError: [Errno 22] invalid mode ('w') or filename: 'c:\nzbToMedia.spec' 

I get the same type of error if I try to compile TorrentToMedia.

Any ideas? I'm not a windows or python guy... But I can cut and paste from the wiki :+1:

**Comment by clinton-hall** [comment]
The first thing on the wiki page is a comment that I can't compile these anymore and instead I now list Python and winPy as a requirements to run these scripts. 

Building .exe files is not practical now as there are too many manual edits required to be able to compile.

**Comment by jobenso** [comment]
I was trying this at work, so maybe I didn't have all the needed components. Python 27 was installed, but winPy might be an issue still. I tried running the script from nzbget and it failed. I agree that compiling this would be a pain, I also you might want to revise the installation instructions to add the requirement for winPy. Honestly I've got this working perfectly in mint, but the problem with linux kernel and mono is preventing me from using sonaar. That's the need for the switch.

General

Install python 2.7.

Clone or copy all files into a directory wherever you want to keep them (eg. /scripts/ in the home directory of your download client) and change the permission accordingly so the download client can access these files.

git clone git://github.com/clinton-hall/nzbToMedia.git
